pages.it/common/git-clean.md

```diff
@@ -0,0 +1,28 @@
+# git clean
+
+> Rimuovi i file non tracciati dall'albero di lavoro.
+> Maggiori informazioni: <https://git-scm.com/docs/git-clean>.
+
+- Rimuovi da Git i file non tracciati:
```

Comment:
```suggestion
- Elimina i file che non sono tracciati da git:
```
Files are not deleted *from* git, but rather from the working tree *if not tracked by* git. Also, rimuovi (remove) -> elimina (delete), more literal.